readme.md

```diff
@@ -168,7 +199,7 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 
 Of the three options below, which is the most "correct" way of organizing the files that make up an Angular app, as used in class? Why is this option considered "better" than the other two?
 
-> Your answer...
+Option C is the most correct because the file names are properly aligned with the folder names. It allows for others to easily locate the files that they need without having to search through many folders.
```

Comment:
C might feel correct because it's very close to rails, but B is more closely aligned with how most angular apps are structured (convention), and how we structured ours in class.